### PR TITLE
DM-2186: Move astrometry_net wrapper code from meas_astrom to a new package

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: config/processCcd.py [hunk 1]
```diff
@@ -23,15 +23,16 @@
     config.charImage.measurePsf.psfDeterminer["psfex"].spatialOrder = 2
     config.charImage.measurePsf.psfDeterminer.name = "psfex"
 except ImportError as e:
-    print "WARNING: Unable to use psfex: %s" % e
```
Comment (parejkoj) [inline config/processCcd.py:26]: How did that get through?

**Review by parejkoj — APPROVED** [review]
Nice catch in there.

File: config/processCcd.py (hunk 1 above)
Comment (r-owen) [inline config/processCcd.py]: Good catch. Thanks!

Comment (r-owen) [reply config/processCcd.py:26]: obs_monocam has no unit tests and has probably never been run under Python 3

Comment (parejkoj) [reply config/processCcd.py:26]: Oh, right... I didn't notice the package name.

**Pull request merged**